src/App.js

```diff
@@ -36,11 +64,17 @@ class App extends Component {
     return (
       <div>
         <div id="appContainer" ref={this.setAppContainerRef}>
-          <LandingHeader onMenuClick={this.onMenuClick} />
+          <Switch>
+            <Route exact path='/' component={this.LandingHeaderWProps}/>
+            <Route path='/check-in' component={this.AppHeaderWProps}/>
+          </Switch>
           <Main />
           <Footer />
         </div>
-        <SideNav />
+        <Switch>
+          <Route exact path='/' component={this.LandingSideNav}/>
+          <Route path='/check-in' component={this.InAppSideNav}/>
```

Comment:
This seems fine for now. If we have to duplicate the paths a bunch more time I think it would make sense to look for a way to avoid that duplication.